### PR TITLE
Update with Amy's new design

### PR DESCRIPTION
**Pull request by khjrtbrg**
- Uses Amy's new design :sparkles:
- Removes Bootstrap (It doesn't seem like we were using it for much other than the grid, so let's go without it!)
- Tidies up some layout inconsistencies between pages
- Converts all templates to haml

**Note:** This PR is pending assets from Amy.

### CODE REVIEW

**Comment by kerrizor** [comment]
Quick surface skim looks great to me!

**Comment by brennx0r** [comment]
Looks awesome!  

Gentle reminder that it needs to include the most recent sponsorship changes that were merged to master earlier today - I don't see the changes present in the branch.  

Thank you so much for doing this!  💟

File: source/index.html.haml [hunk 1]
```diff
@@ -1,46 +1,36 @@
...
-    %p Open Source & Feelings is about the intersection between software and the humanities, and how we engage with the communities we're a part of. It is about deliberative crafting of its culture into what we want it to be, about developing strategies, skills, and solutions in order work as a community.
+.index
+  .row
+    .col-3
+      .item
+        %h2.teal.text-left Why Open Source?
+        %p While successful from a technical or business perspective, on the human side of things, the promise of the Open Source Software community has gone unfulfilled. Open Source culture, left to grow untended, has been co-opted, diluted, and leaves too many individuals behind. It is a garden in need of tending, with too few gardeners available—we want to change that.
+      .item
+        %h2.yellow.text-left Why Feelings?
+        %p Writing software can be emotionally difficult. Some of us talk about it with others, but many of us don’t, or aren’t encouraged to share. We need to be sharing our struggles, however, as we are the most natural support for each other. Being supportive and using this support takes practice and it takes intention.
```
Comment (brennx0r) [inline source/index.html.haml]: Would it make sense to have an addition %p here?  This way, the paragraph alignment with the column to the left and right will be equal. Right now, the paragraphs are out of alignment with the other column paragraphs. I'm not 100% that a simple additional %p would help here.

**Comment by brennx0r** [comment]
I would love to see the Where and When information on main page move up above the "Why?" columns, with the conference date and place information embigginated.

**Comment by khjrtbrg** [comment]
@brennx0r I pulled in the sponsor page changes and moved the Where and When section to the top. Let me know what you think! 💖

**Comment by brennx0r** [comment]
The Where&When is looking awesome!

A couple more issues to resolve, and then I think this will be ready to merge.  

1.  Using Chrome, head over to the sponsors page and take a look at the padding between the columns for the in-kind sponsors.  It looks like there's no padding between the columns. :(

2.  Go to any of the following pages:

http://localhost:4567/previous-speakers/2015/casey-west
http://localhost:4567/previous-speakers/2015/isaac-z.-schlueter
http://localhost:4567/previous-speakers/2015/taylor-dewey

It looks like a combination of the column width, image width, and the title text sometimes creates a situation where the title text is presented to the right of the image, instead of below the image. It would be preferable that all the pages have the following resultant formatting:

http://localhost:4567/previous-speakers/2015/rachel-ober

Thanks again for taking this on! These changes are super exciting!

**Comment by khjrtbrg** [comment]
@brennx0r Whoops, I didn't intend for the individual speaker page to look like that! Can you take a peek and see how it looks now?

As for the sponsors page padding, I'm not seeing it on my end. Can you take another look and see if it's still happening for you? 💖

**Comment by brennx0r** [comment]
Lookin' awesome!  :shipit:

**Pull request merged**